### ISSUE TITLE
Absolute paths in systemjs config are not moved to /base

### ISSUE THREAD

**Issue by youngbob** [rolaveric/karma-systemjs#53]
If your systemjs config contains absolute paths, fixing the baseURL will not also fix these paths. They also need to be relocated under /base, where Karma serves them.

```
System.config({
  baseURL: '/app',
  map: {
    'jquery': '/thirdparty/jquery.js'
  }
});
```

While these paths could be overridden in the Karma config, this becomes difficult to maintain if your config contains a large number of frequently changing thirdparty paths.

**Comment by rolaveric** [comment]
Fixed by #54 which has been merged into master and released with 0.8.1

**Comment by rolaveric** [comment]
No it hasn't - I was looking at #52 ...

**Comment by rolaveric** [comment]
OK, now #54 has been merged into master, and released with 0.8.2
